parlai/agents/transformer/transformer.py

```diff
@@ -103,12 +131,25 @@ def score_candidates(self, batch, cand_vecs):
         else:
             mems = None
 
-        return self.model(
+        context_h, cands_h = self.model(
```

Comment:
Not sure what motivated moving those 2 guys outside self.model() but that's interesting for this project:
https://github.com/facebookresearch/ParlAI/pull/1416

Reply:
We needed to do this to calculate the scores in data parallel mode